src/components/message/message-card/MessageCard.tsx

```diff
@@ -77,87 +76,91 @@ function MessageCardBase({ message, showRaw, showRawHandler }: Props) {
+					<div className='mc-header__info'>
+						<div className='mc-header__value'>
+							{timestamp && formatTime(getTimestampAsNumber(timestamp))}
+						</div>
+						<div className='mc-header__item'>
+							<span className='mc-header__key-minified'>nm</span>
+							<span className='mc-header__key'>Name</span>
+							<span className='mc-header__value'>{messageType}</span>
+						</div>
+						<div className='mc-header__item'>
+							<span className='mc-header__key-minified'>ss</span>
+							<span className='mc-header__key'>Session</span>
+							<span className={sessionClass} style={sessionArrowStyle}></span>
+							<span className='mc-header__value'>{sessionId}</span>
+						</div>
+						<div className='mc-header__item'>
+							<span className='mc-header__key-minified'>id</span>
+							<span className='mc-header__key'>ID</span>
+							<span className='mc-header__value'>{messageId}</span>
+						</div>
 					</div>
-				)}
-				<div className='mc-header__pin'>
 					<div
-						title={isPinned ? 'Unpin' : 'Pin'}
-						className={pinClassName}
-						onClick={() => selectedStore.toggleMessagePin(message)}
-					/>
+						className={bookmarkIconClass}
+						title={isPinned ? 'Remove from bookmarks' : 'Add to bookmarks'}
+						onClick={() => selectedStore.toggleMessagePin(message)}></div>
 				</div>
-				<div className={underlineClassName} />
-			</div>
-			<div className='message-card__body mc-body'>
-				<div className='mc-body__human'>
-					<ErrorBoundary
-						fallback={
-							<MessageBodyCardFallback
+				<div className='mc__mc-body mc-body'>
+					<div className='mc-body__human'>
+						<ErrorBoundary
+							fallback={
+								<MessageBodyCardFallback
+									isBeautified={isContentBeautified}
+									isSelected={color !== undefined}
+									body={body}
+								/>
+							}>
+							<MessageBodyCard
 								isBeautified={isContentBeautified}
-								isSelected={color !== undefined}
 								body={body}
+								isSelected={color !== undefined}
 							/>
-						}>
-						<MessageBodyCard
-							isBeautified={isContentBeautified}
-							body={body}
-							isSelected={color !== undefined}
-						/>
-					</ErrorBoundary>
-					{bodyBase64 && bodyBase64 !== 'null' ? (
-						<div className='mc-show-raw' onClick={() => showRawHandler(!showRaw)}>
-							<div className='mc-show-raw__title'>RAW</div>
-							<div className={showRawClass} />
-						</div>
-					) : null}
-					{bodyBase64 && showRaw ? (
-						<MessageRaw messageId={messageId} rawContent={bodyBase64} />
-					) : null}
+						</ErrorBoundary>
+						{bodyBase64 && showRaw ? (
+							<MessageRaw messageId={messageId} rawContent={bodyBase64} />
+						) : null}
+					</div>
+					<div className='mc-body__controls'>
+						{message.body !== null && (
+							<div
+								className={beautifyButtonClass}
+								onClick={() => messagesStore.toggleMessageBeautify(messageId)}
+							/>
+						)}
+						{bodyBase64 && bodyBase64 !== 'null' ? (
```

Comment:
bodyBase64 value is compared to a string. Are you sure it's ok?

Reply:
it's not my code fragment, i just moved it to new layout without any changes. so, idk